wrappers/services.go

```diff
@@ -1804,7 +1808,7 @@ type RestartServicesFlags struct {
 // TODO: change explicitServices format to be less unusual, more consistent
 // (introduce AppRef?)
 func RestartServices(svcs []*snap.AppInfo, explicitServices []string,
-	flags *RestartServicesFlags, inter Interacter, tm timings.Measurer) error {
+	flags RestartServicesFlags, inter Interacter, tm timings.Measurer) error {
```

Comment:
this change is a bit strange because now StopService and StartServices are inconsistent, I would leave things as they were

Reply:
I've reverted this part. In general I would prefer to force having to pass the parameter as it is more explicit and avoids checks for nil, but as things are the same for `StopServices` I guess that should be its own PR.